lib/vparse.c

```diff
@@ -1032,7 +1033,9 @@ EXPORTED void vparse_replace_entry(struct vparse_card *card, const char *group,
     }
 }
 
-EXPORTED void vparse_delete_entries(struct vparse_card *card, const char *group, const char *name)
+static void _delete_entries(struct vparse_card *card,
```

Comment:
I wonder if the label counting code in here should be moved to its own function `count_apple_labels`. Saving one iteration of the cards properties doesn't seem to warrant the mental overhead that `_delete_entries` really does not just delete entries anymore.

Reply:
This is going to be rolled into one function per the comment below